arbitrum-docs/build-decentralized-apps/token-bridging/layer-leap.mdx

```diff
@@ -0,0 +1,84 @@
+
+- Bridging `ETH` requires one call on L1
+- Bridging an `ERC-20` requires one approval and one call on L1
+
+**Orbit L3s:** Can more seamlessly onboard `ETH` and `ERC-20`s from Ethereum mainnet.
+
+_Which Orbit configurations can adopt this feature?_
+
+**Any L3 Orbit Chains on top of an Arbitrum chain** **_with ETH gas fees_** can use Layer Leap.
+
+- Layer Leap contracts are currently only deployed for Arb One and Arb Nova, but contracts can be deployed for additional Arbitrum Layer-2s by Offchain Labs or your RaaS provider.
+  - The Layer 2 must be an Arbitrum chain due to the use of retryables.
+- _Future support: Layer Leap will support Orbit chains with custom gas tokens in a future release_
+
+## Implementation
+
+_How can Orbit chains adopt this feature?_
+
+**UI Adoption:** Layer leap contracts can be integrated into your own bridge or added to your chain’s UI for the Arbitrum bridge (bridge.arbitrum.io)
+
+### Arbitrum bridge UI: if you are already on the arbitrum bridge UI then you can sign up for layer leap adoption by using this [github template](https://github.com/offchainlabs/arbitrum-token-bridge/issues/new?assignees=&labels=feat%2ctriage&projects=&template=add-layerleap-request.yml&title=%5bfeat%5d%3a+enable+layer+leap+for+%3corbit+chain%3e).
+
+    - If you are not on the Arbitrum Bridge UI yet you can sign up by filling out this [GitHub Template](https://github.com/OffchainLabs/arbitrum-token-bridge/issues/new?assignees=&labels=feat%2Ctriage&projects=&template=add-orbit-chain-request.yml&title=%5Bfeat%5D%3A+), and then also fill out the template above.
+    - You can demo Layer Leap on the Arbitrum Bridge by bridging to [Rari](https://bridge.arbitrum.io/?destinationChain=rari-mainnet&sourceChain=ethereum) and [Proof of Play](https://bridge.arbitrum.io/?destinationChain=pop-apex&sourceChain=ethereum).
+
+### Custom bridge UI: if you want to integrate layer leap functionality into your custom bridge UI, read the [info page on the layer leap github](https://github.com/offchainlabs/l1-l3-teleport-contracts/blob/main/docs/info.md) to learn more.
+
+_What chains are currently supported?_
+
+Layer Leap requires an L1 teleporter contract for each L2 chain. `L1Teleporter` contracts are deployed on Ethereum L1 for Arb One and Arb Nova.
+
+| Contract              | Address                                                                 |
+| --------------------- | ----------------------------------------------------------------------- |
+| Arb One L1Teleporter  | https://etherscan.io/address/0xCBd9c6e310D6AaDeF9F025f716284162F0158992 |
+| Arb Nova L1Teleporter | https://etherscan.io/address/0xCBd9c6e310D6AaDeF9F025f716284162F0158992 |
+
+For non Arbitrum One L3s:
+
+- To enable Layer Leap for your Layer 3 you can follow the Deployment Procedure defined [here](https://github.com/OffchainLabs/l1-l3-teleport-contracts/blob/main/docs/info.md). Before self deploying, let Offchain Labs know about your request!
```

Comment:
What about for L3s on Nova? It's not clear to me without reading a bit more that a new contract deployment is what is required for L3s on Arb1 and Nova. 

Also how does someone contact offchain labs here? We should add a contact link/form.